Author: Vincent Legout <vincent@legout.info>
Bug-Debian: http://bugs.debian.org/580711
Description: Fix a segmentation fault due to missing strings in language files

Index: billard-gl-1.75/lp/901.lang
===================================================================
--- billard-gl-1.75.orig/lp/901.lang	2010-05-08 22:19:14.075833001 +0800
+++ billard-gl-1.75/lp/901.lang	2010-05-08 22:19:43.659833001 +0800
@@ -143,6 +143,8 @@
 295 Texture Interpolation :
 296 Off
 297 Linear
+298 Standard
+299 High
 
 300 Quality :
 301 Very Fast
Index: billard-gl-1.75/lp/904.lang
===================================================================
--- billard-gl-1.75.orig/lp/904.lang	2010-05-08 22:20:00.947833001 +0800
+++ billard-gl-1.75/lp/904.lang	2010-05-08 22:20:33.283833001 +0800
@@ -141,6 +141,8 @@
 295 Interpolacion de texturas :
 296 No
 297 Lineal
+298 Normal
+299 Alta
 
 300 Calidad :
 301 Muy rápido
Index: billard-gl-1.75/lp/905.lang
===================================================================
--- billard-gl-1.75.orig/lp/905.lang	2010-05-08 22:21:10.631833001 +0800
+++ billard-gl-1.75/lp/905.lang	2010-05-08 22:21:48.847833001 +0800
@@ -141,6 +141,8 @@
 295 Struktur Interpolation :
 296 Off
 297 Lineær
+298 Standard
+299 High
 
 300 Kvalitet :
 301 Meget Hurtig
Index: billard-gl-1.75/lp/907.lang
===================================================================
--- billard-gl-1.75.orig/lp/907.lang	2010-05-08 22:21:19.331833000 +0800
+++ billard-gl-1.75/lp/907.lang	2010-05-08 22:21:40.583833001 +0800
@@ -141,6 +141,8 @@
 295 Struktur Interpolation :
 296 Off
 297 Lineær
+298 Standard
+299 High
 
 300 Kvalitet :
 301 Meget Hurtig
